spec/venue_spec.rb

```diff
@@ -44,12 +42,48 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe '#over_capcity' do
+    it 'returns true based on capacity limit' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('Bobby')
+      venue.add_patron('Robert')
+      expect(venue.over_capacity?).to be(true)
+    end
+  end
+
+  describe '#over_capcity checker' do
+    it 'returns false based on under capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      expect(venue.over_capacity?).to be(false)
+    end
+  end
+
+  describe '#kick_out' do
+    it 'remove patrons until capacity met' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      venue.kick_out
+      expect(venue.over_capacity?).to be(false)
```

Comment:
Great test! Maybe something else you could add here is a check for the specific number of patrons you expect to be in the patrons array.